src/solana/SolanaEngine.ts

```diff
@@ -746,6 +765,7 @@ export class SolanaEngine extends CurrencyEngine<
       instructions.push(memoInstruction)
     }
 
+    if (this.recentBlockhash === '') await this.queryBlockhash()
```

Comment:
It seems like we are only going to load the block hash once at boot now. Wouldn't that make it even more stale?

Based on the commit message, I would have expected `startEngine` to ignore the blockhash entirely, and then `signTx` to call it anew for each signature, perhaps with a `Date.now()` timestamp so we can skip the query if we have a value from the last X seconds ago.